src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,29 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer volvo = new Manufacturer();
+        volvo.setName("Volvo");
+        volvo.setCountry("China");
+        System.out.println("Creating new manufacturer Volvo" + manufacturerDao.create(volvo));
+        System.out.println("Getting one of existing manufacturers from DB "
+                + manufacturerDao.get(3L));
+        System.out.println("Deleting all one of existing manufacturers from DB "
+                + manufacturerDao.delete(8L));
+        Manufacturer manufacturerUpdate = new Manufacturer();
+        manufacturerUpdate.setId(10L);
+        manufacturerUpdate.setName("Seat");
+        manufacturerUpdate.setCountry("France");
+        System.out.println("Updating existing manufacturer with id 10 "
+                + manufacturerDao.update(manufacturerUpdate));
```

Comment:
Let's give more common names for variables, like "seat".

You need to create the Manufacturer before update.
```suggestion
        Manufacturer seat = new Manufacturer();
        seat.setName("Seat");
        seat.setCountry("France");
        manufacturerDao.create(seat);
        seat.setName("SEAT S.A.");
        System.out.println("Updating existing manufacturer: "
                + manufacturerDao.update(seat));
```